### ISSUE TITLE
Булгаков, "Дни Турбиных"

### ISSUE THREAD

**Issue by yaskevich** [dracor-org/rusdracor#13]
In act 4 ("Действие четвертое") there are no scenes inside in _bulgakov-dni-turbinyh.xml_

**Comment by lehkost** [comment]
Yes, but there's nothing we can do about it. What do you suggest?

**Comment by yaskevich** [comment]
Perhaps, it has to be treated in the same manner as plays without internal act/scene separation, when we add sections for logical parts of a piece of writing.
It seems to be better solution than leave it like this, when we have the acts with scenes and only act that is just act and holds text content. 
The more so such a structure doesn't render  a "canon", as long as author's vision regarding the editions of the play is questionable.
